.gitignore

```diff
@@ -1,3 +1,5 @@
+# vim
+*.swp
```

Comment:
Do you mind moving this down to the bottom with the organized IDE/Editor ignores? The top part is pretty ugly.

Reply:
done